src/status_im/ui/screens/rpc_usage_info.cljs

```diff
@@ -98,35 +106,36 @@
           (str k " " v))
         stats)))
 
-(defn usage-info []
-  (let [stats @(re-frame/subscribe [:rpc-usage/data])
-        methods-filter @(re-frame/subscribe [:rpc-usage/filter])]
-    [react/view {:flex 1
-                 :margin-horizontal 8}
-     [quo.react-native/view
-      {:style {:flex-direction  :row
-               :margin-top      8
-               :justify-content :space-between}}
-      [quo.core/button
-       {:on-press            #(re-frame/dispatch [::get-stats])
-        :accessibility-label :rpc-usage-get-stats}
-       (i18n/label :t/rpc-usage-get-stats)]
-      [quo.core/button
-       {:on-press            #(re-frame/dispatch [::reset])
-        :accessibility-label :rpc-usage-reset}
-       (i18n/label :t/rpc-usage-reset)]
-      [quo.core/button
-       {:on-press
-        #(react/copy-to-clipboard (prepare-stats stats))
-        :accessibility-label :rpc-usage-copy}
-       (i18n/label :t/rpc-usage-copy)]]
-     [quo.core/text-input
-      {:on-change-text  #(re-frame/dispatch [::set-filter %])
-       :label           (i18n/label :t/rpc-usage-filter)
-       :default-value   methods-filter
-       :auto-capitalize :none
-       :show-cancel     false
-       :auto-focus      false}]
-     [stats-table stats]]))
-
+(defview usage-info []
+  (letsubs []
+    {:component-did-mount (fn []
+                            (reset! rpc-refresh-interval (utils/set-interval #(re-frame/dispatch [::get-stats]) rpc-usage-refresh-interval-ms)))
+     :component-will-unmount (fn []
+                               (utils/clear-interval @rpc-refresh-interval)
+                               (reset! rpc-refresh-interval nil))}
 
+    (let [stats @(re-frame/subscribe [:rpc-usage/data])
```

Comment:
if you used letsubs , you need to move subsciptions to it, and also i think map with lifecycle methods must be outside of letsubs

Reply:
It's a small hack to make sure the UI is updated when we dispatch the get-stats method. Apologies, I'll create a container and push it.